airbyte-integrations/connectors/source-mailgun/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_mailgun'
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:base-python').airbyteDocker.outputs)
```

Comment:
```suggestion
```

Reply:
If I know correctly, then when testing using the `gradle` command, we pull off the last image of SAT, and therefore we generally do not need to declare any `dependencies` in this file.